<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>War_Limit</class>
 <widget class="QWidget" name="War_Limit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="3,5">
   <item row="0" column="0" rowspan="6">
    <widget class="QTableView" name="tableView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color :rgb(250,250,250);</string>
     </property>
     <property name="lineWidth">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>告警信息</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="styleSheet">
      <string notr="true">background-color :rgb(250,250,250);
color :rgb(255,0,0);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>限位信息</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="styleSheet">
      <string notr="true">background-color :rgb(250,250,250);
color :rgb(0,0,255);</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>版本显示</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
